packages/web/components/Player/ColorBar.tsx

```diff
@@ -57,25 +65,34 @@ export const ColorBar = ({
           );
         })}
       </Flex>
-      <Flex minH="calc(1.5rem + 4px)" maxW="100%" border="2px" borderRadius={3}>
+      <Flex
+        minH="calc(1.5rem)"
```

Comment:
```suggestion
        minH="1.5rem"
```

Reply:
With `calc()`, this is fine but when i remove it and just have `"1.5rem"` it throws a formatting error. 🤷🏻  
![image](https://user-images.githubusercontent.com/780350/121032471-348a2f00-c7a3-11eb-89f5-d1a69dccff2f.png)

![image](https://user-images.githubusercontent.com/780350/121032410-29cf9a00-c7a3-11eb-8b73-3e652fa234c9.png)

Reply:
@luxumbra  you shouldn't have a semi colon there